src/VennDiagram/VennArc.tsx

```diff
@@ -172,6 +172,8 @@ export const VennArc: FC<Partial<VennArcProps>> = ({
     }
   });
 
+  const tooltipData = useMemo(() => ({ y: data.data.size, x: data.data?.sets?.join(' | ') }), [data]);
```

Comment:
Same https://github.com/reaviz/reaviz/pull/182/files#r1437926574

Reply:
This is not the same as https://github.com/reaviz/reaviz/pull/182/files#r1437926574. It's just moving the old tooltip value code into a variable for it to be reused.